src/client.ts

```diff
@@ -230,11 +242,13 @@ export class Client {
      *
      */
     public async call(api: string, method: string, params: any[] = []): Promise<any> {
+        let callApi = api
+        if (this.useAppbaseApi) callApi = 'condenser_api'
```

Comment:
This naive approach will break programs that use APIs other than `condenser_api`

Reply:
This is a much better approach https://github.com/steemit/dsteem/pull/3

Will merge that instead

Reply:
I think all appbase APIs are available with condenser_api
Anyway, I think steemit#3 will work too

Reply:
I've recently rewritten my code with client.call('condenser_api', ....).

With 0.10.0, this seems to be obsolete, right?

Reply:
That's right. I'm closing this. Thanks for the PR nevertheless @mahdiyari